CMakeLists.txt

```diff
@@ -53,6 +54,19 @@ else(MSVC)
   endif()
 endif(MSVC)
 
+if(USE_NNPACK)
+  find_package(NNPACK REQUIRED)
+  if(NNPACK_FOUND)
+    message(STATUS "Found NNPACK: ${NNPACK_INCLUDE_DIR}")
+    add_definitions(-DMXNET_USE_NNPACK=1)
+    include_directories(SYSTEM ${NNPACK_INCLUDE_DIR})
+    include_directories(SYSTEM ${NNPACK_INCLUDE_DIR}/../third-party/pthreadpool/include)
+    list(APPEND REQUIRED_LIBRARIES ${NNPACK_LIB})
+  else()
+    message(FATAL_ERROR "Can't find NNPACK. Please set NNPACK_FOUND, NNPACK_INCLUDE_DIR and NNPACK_LIB")
```

Comment:
i think user don't need set NNPACK_FOUND. and your cmake script is expecting a NNPACK_ROOT